src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Test;
+
+
+public class UtilsTest {
+
+    @Test
+    public void nullObject_returnsTrue() {
+        final Object[] inputs = { "", null, "\n  \n" };
+        final boolean resultMessage = true;
+        assertEquals(resultMessage, Utils.isAnyNull(inputs));
+    }
+    
+    @Test
+    public void noNullObject_returnsFalse() {
+        final Object[] inputs = { "", "  ", "\n  \n" };
+        final boolean resultMessage = false;
+        assertEquals(resultMessage, Utils.isAnyNull(inputs));
+    }
+    
+    @Test
+    public void checkUnique_returnsTrue() {
+        final List<String> inputs = Arrays.asList("A", "B", "C", "D");
+        final boolean resultMessage = true;
+        assertEquals(resultMessage, Utils.elementsAreUnique(inputs));
+    }
+    
+    @Test
+    public void checkUnique_returnsFalse() {
+        final List<String> inputs = Arrays.asList("A", "B", "C", "D", "A");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.